lib/rules/no-yield-to-default.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'You should not yield to default';
+
+const DISALLOWED_PATHS = ['yield', 'has-block', 'has-block-params', 'hasBlock', 'hasBlockParams'];
+
+module.exports = class NoYieldToDefault extends Rule {
+  visitor() {
+    return {
+      MustacheStatement(node) {
+        if (DISALLOWED_PATHS.includes(node.path.original)) {
```

Comment:
We can't assume that `.path.original` is present. For example with `{{"foo"}}` there is no `.path.original` (I think its `.path.value`?).

Reply:
But we are looking for `{{yield}}` and `{{has-block}}` nodes right, for them the `path.original` is always there.

Reply:
> But we are looking for {{yield}} and {{has-block}} nodes right, for them the path.original is always there.

This is no longer correct, `node.path` is allowed to be roughly "anything". For example, in Ember 3.25+ this is perfectly valid:

```hbs
{{(call-some-helper-that-returns-a-helper) whatever-here="lol"}}
```